xla/service/gpu/gpu_offloading_test.cc

```diff
@@ -132,10 +136,51 @@ ENTRY %main (param_0: f32[1024], param_1: f32[1024]) -> f32[1024] {
   ROOT %res_11 = f32[1024]{0} tanh(f32[1024]{0} %res_10)
 }
 )";
-  EXPECT_TRUE(RunAndCompare(hlo_text, ErrorSpec{/*aabs=*/1e-6, /*arel=*/1e-6}));
```

Comment:
Can you please update this to EXPECT_GT? This will block the merge.

Reply:
This version is out of date and it shouldn't appear. I squashed the 9 commits into 1 to solve this merge issue.

Reply:
I am still getting an error for the EXPECT_TRUE function, the line I'm getting in the error does not correspond to any usage in the PR. Would appreciate some help figuring out where the issue is. This is blocking the merge. The exact error is "Consider using EXPECT_GT(a, b) instead of EXPECT_TRUE(a > b)"

Reply:
Thanks @sgerrard. That's my bad. Changed to EXPECT_GT().